### ISSUE TITLE
Multiple dex files define FinalizeableReference

### ISSUE THREAD

**Issue by samtstern** [googlesamples/google-services#321]
Unable to update the `signin` sample to `11.0.0` because of this error:

```
* What went wrong:
Execution failed for task ':app:transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lcom/google/common/base/FinalizableReference;
```

**Comment by bitsydarel** [comment]
Same here...

Error:Error converting bytecode to dex:
Cause: com.android.dex.DexException: Multiple dex files define Lcom/google/common/base/FinalizableReference;

**Comment by bitsydarel** [comment]
for 
com.google.android.gms:play-services-location

**Comment by dakdroid** [comment]
Same for me.

`//FIREBASE
compile 'com.google.firebase:firebase-core:11.0.0'
compile 'com.google.firebase:firebase-analytics:11.0.0'
compile 'com.google.firebase:firebase-crash:11.0.0'
compile 'com.google.firebase:firebase-invites:11.0.0'
compile 'com.google.firebase:firebase-messaging:11.0.0'
compile 'com.google.firebase:firebase-config:11.0.0'
compile 'com.google.firebase:firebase-ads:11.0.0'

//Playservices:
compile 'com.google.android.gms:play-services-base:11.0.0'
compile 'com.google.android.gms:play-services-auth:11.0.0'
compile 'com.google.android.gms:play-services-cast:11.0.0'`

**Comment by samtstern** [comment]
@bitsydarel @dakdroid we're working on this, we will have a solution soon.  Thanks for your patience!

**Comment by enricoros** [comment]
Same here:
    def gmsVer = '11.0.0'
    compile "com.google.android.gms:play-services-auth:$gmsVer"
    compile "com.google.android.gms:play-services-drive:$gmsVer"
    compile "com.google.firebase:firebase-auth:$gmsVer"
    compile "com.google.firebase:firebase-database:$gmsVer"

**Comment by samtstern** [comment]
This is closed with `11.0.1`

**Comment by talhakosen** [comment]
I get below error for all version 11.0.1

Error:Failed to resolve: com.google.firebase:firebase-core:11.0.1
<a href="openFile:/Repo/bckspace/transparent/transparent-android/app/build.gradle">Open File</a><br><a href="open.dependency.in.project.structure">Show in Project Structure dialog</a>
